### PR TITLE
[Artur Kostiuk]Product details first review

### PR DESCRIPTION
**Pull request by art1kos**

### CODE REVIEW

**Comment by dverbovyi** [comment]
@taras1313 please review the request and provide your feedback

File: src/app/product-details-page/components/product-description/product-description.component.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <div class="main-container-wrapper">
+      <div class="image-container">
+      </div>
+      <div class="details-container">
+        <div class="details-container-info">
+          <h2>{{productDetails.title}}</h2>
+          <h5>Half Jacket + Skiny Trousers + Boot leather</h5>
+          <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry Lorem Ipsum is simply dummy text
+            of the printing and typesetting industry</p>
+        </div>
```
Comment (taras1313) [inline src/app/product-details-page/components/product-description/product-description.component.html]: split by new line

File: src/app/product-details-page/components/product-description/product-description.component.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<div *ngIf="productDetails">
+  <div class="main-container">
+    <div class="main-container-wrapper">
+      <div class="image-container">
+      </div>
+      <div class="details-container">
```
Comment (taras1313) [inline src/app/product-details-page/components/product-description/product-description.component.html:5]: split by new line

**Review by taras1313 — DISMISSED** [review]
minor fixes, and it is ready for merge

File: src/app/product-details-page/components/product-description/product-description.component.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+export class ProductDescriptionComponent implements OnInit {
+
+  productDetails: IProduct;
+  constructor(
+    private activatedRoute: ActivatedRoute,
+    private productsService: ProductsService
+  ) { }
+
+  ngOnInit() {
+    const id: string = this.activatedRoute.snapshot.params.id;
```
Comment (dverbovyi) [inline src/app/product-details-page/components/product-description/product-description.component.ts]: it should be done in the page-level component

File: src/app/product-details-page/components/product-details-page/product-details-page.component.html [hunk 4]
```diff
@@ -0,0 +1 @@
+<app-product-description></app-product-description>
```
Comment (dverbovyi) [inline src/app/product-details-page/components/product-details-page/product-details-page.component.html]: product-description should receive title (product title) and description (product description itself) attributes

**Review by dverbovyi — CHANGES_REQUESTED** [review]

File: src/app/product-details-page/components/product-description/product-description.component.html [hunk 5]
```diff
@@ -0,0 +1,16 @@
+  <div class="main-container">
+    <div class="main-container-wrapper">
+      <div class="image-container"></div>
+
+      <div class="details-container">
+        <div class="details-container-info">
+          <h2>{{title}}</h2>
+          <h5>{{description}}</h5>
+          <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry Lorem Ipsum is simply dummy text
```
Comment (dverbovyi) [inline src/app/product-details-page/components/product-description/product-description.component.html]: should be retrieved from product model along with title etc

**Review by dverbovyi — CHANGES_REQUESTED** [review]

File: src/app/product-details-page/components/product-description/product-description.component.ts [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+  selector: 'app-product-description',
+  templateUrl: './product-description.component.html',
+  styleUrls: ['./product-description.component.scss']
+})
+export class ProductDescriptionComponent implements OnInit {
+  @Input() productTitle;
+  @Input() productDescription;
+  @Input() productName;
+
+  name: string;
```
Comment (dverbovyi) [inline src/app/product-details-page/components/product-description/product-description.component.ts]: remove all this stuff together with ngOnInit

File: src/app/product-details-page/components/product-description/product-description.component.ts [hunk 7]
```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-product-description',
+  templateUrl: './product-description.component.html',
+  styleUrls: ['./product-description.component.scss']
+})
+export class ProductDescriptionComponent implements OnInit {
+  @Input() productTitle;
```
Comment (dverbovyi) [inline src/app/product-details-page/components/product-description/product-description.component.ts]: rename 

<app-product-description [name]="product.name" [title]="product.title" [description]="product.description"

@Input() name: string;
@Input() title: string;
@Input() description: string; (edited)

**Review by dverbovyi — CHANGES_REQUESTED** [review]

**Review by dverbovyi — APPROVED** [review]
add unit tests with the next PR

**Pull request merged**